common/crypto/openssl/cert.c

```diff
@@ -651,8 +657,10 @@ oe_result_t oe_cert_chain_read_pem(
         tmp_pem_data[pem_size] = '\0';
     }
 
+#if !defined(OE_BUILD_ENCLAVE)
     /* Initialize OpenSSL (if not already initialized) */
-    oe_initialize_openssl();
+    oe_crypto_initialize();
```

Comment:
How can we be sure that we are not missing any control path that requires initialization?

Reply:
Currently implementation retains the behavior on the host (lazy initialization), but explicitly does the initialization on the enclave side.

Reply:
In the future, I'd like to see this unified so the behavior is the same on the host and the enclave so all these ifdefs can go away.

Reply:
Once we drop the ubuntu 16.04 (which supports older version of OpenSSL), we will be able to remove these code.